README.md

```diff
@@ -211,6 +211,42 @@ Below is a list of available tabs in a GDPR privacy manager:
     }
 ```
 
+# GetSpConsent
+
+Getter for SpConsents data. After calling, checks platform(Android or ios) and return SPConsents structure:
+
+```c#
+    SpConsents
+        |-- gdpr?
+        |   |-- applies: bool
+        |   |-- consents: GdprConsent
+        |       |-- uuid: String?
+        |       |-- tcData: Map<String, String>
+        |       |-- grants: Map<String, GDPRPurposeGrants>
+        |       |-- euconsent: String
+        |       |-- acceptedCategories: List<String>
+        |       |-- webConsentPayload: String
+        |       |-- consentStatus: ConsentStatus
+        |-- ccpa?
+            |-- applies: bool
+            |-- consents: CcpaConsent
+                |-- uuid: String?
+	        |-- rejectedCategories: List<String>
+                |-- rejectedVendors: List<String>
+                |-- status: String?
+                |-- uspstring: String
+                |-- childPmId: String
+                |-- signedLspa: bool
+                |-- webConsentPayload: String
```

Comment:
I believe we should omit `webConsentPayload` on both GDPR and CCPA, as it's intended to be used internally only.